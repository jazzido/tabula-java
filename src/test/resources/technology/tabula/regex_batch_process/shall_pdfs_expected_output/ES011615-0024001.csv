WAKE CO ENVIRONMENTAL HEALTH,,DAVCIA LADEGAARD,
P O BOX 550,,,
"",,2801 DOVER FARM RD,
"RALEIGH, NC 27602 Courier # 51-91-00",,"RALEIGH, NC 27606",
EIN: 566000347EH,,,
StarLiMS ID: ES011615-0024001 Date Collected:,01/15/15,Time Collected:,1:00 PM
Date Received:,01/16/15,Collected By:,
Sample Type: Sampling Point:,well,Well Permit #:,
Analyte,,Result,Allowable Limit,Unit Qualifier(s)
Uranium,,< 0.001,0.030,mg/L
Report Date:,01/23/2015,,Reported By:,Arnold Hall
