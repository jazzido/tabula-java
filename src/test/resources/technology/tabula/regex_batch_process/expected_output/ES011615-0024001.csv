Uranium,< 0.001,0.030,mg/L
